lighthouse/helpers/plates.py

```diff
@@ -263,6 +286,118 @@ def send_to_ss_heron_plates(body: Dict[str, Any]) -> requests.Response:
         raise requests.ConnectionError("Unable to access Sequencescape")
 
 
+# TODO (DPL-572) move this class to another file?
```

Comment:
Should this be done or the TODO removed?  The reference in this TODO is for the ticket this PR is addressing.  If it needs thinking about, please create another ticket and reference that.

Reply:
Checked and removed all DPL-572 TODO comments.